tools/autograd/templates/VariableType.h

```diff
@@ -1,7 +1,7 @@
 #pragma once
 
 // ${generated_comment}
-
+#include <Python.h>
```

Comment:
You don't need to do this now.

In more details, we added a header `torch/csrc/utils/python_stub.h` which just declares `PyObject`. So now, most files which previously included a `Python.h` no longer do so, so it should be necessary to stick `Python.h` on a header unless it really truly is related to the Python binding code.

Reply:
How do I fix these warnings then?

Reply:
What’s the problem with including Python.h? python_stub.h is a super ugly hack

Reply:
@zdevito I think if you delete the Python include, the warnings will be gone now. If this is not the case, find the header that is still including `Python.h` and replace it with an include of python_stub.

@apaszke The problem with Python.h is that it is infectious. If you add Python.h to a "non-Python" header (like `variable.h`) just because you needed to declare `PyObject*` reference, now EVERY cpp file which transitively imports this header must paste a `Python.h` reference to be warnings clean. This is really awful.

Reply:
So all of these Python.h placements are to remove warnings I currently get when building (python 2). Is there another way of going about removing them?